src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[3];
+        for (Ball ball : balls) {
+            ball = lottery.getRandomBall();
+            System.out.println(ball.toString());
```

Comment:
There is no need to call `toString()` explicitly when printing an object. The `println` method calls the `toString()` method of the object internally if it's not a string. You can simply use `System.out.println(ball);`.